### ISSUE TITLE
File gets flagged as a virus

### ISSUE THREAD

**Issue by JohnNielse** [juby-httptoolkit/httptoolkit-desktop-go#1]
https://www.virustotal.com/gui/file/c136a38308de88f2c753a66a86026c0589f1ac1831f37fe6559cb49761507929/detection
.exe file gets flagged and deleted by Windows Defender

**Comment by Juby210** [comment]
I don't think that I can do anything about that.. It is just garbage AVs detecting every unknown executable as malicious. Fell free to check code and build actions or even build yourself if you want
